src/skills-builder/skills-builder-modal/select-preferences/JobTitleAutosuggest.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form,
+} from '@edx/paragon';
+import { logError } from '@edx/frontend-platform/logging';
+import { useAlgoliaSearch } from '../../utils/search';
+
+const JobTitleAutosuggest = ({ onSelected }) => {
+  // We only need to destructure the job search index here
+  const [,, jobSearchIndex] = useAlgoliaSearch();
+
+  const [jobInput, setJobInput] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [jobList, setJobList] = useState([]);
+
+  const queryForJobs = (query = '') => {
+    jobSearchIndex.search(query).then(({ hits }) => {
+      setJobList(hits);
+      setIsLoading(false);
+    }).catch(err => {
+      logError(err);
+      setJobList([]);
+      setIsLoading(false);
+    });
+  };
+
+  const handleAutoSuggestChange = (title) => {
+    setIsLoading(true);
+    setJobInput(title);
+    queryForJobs(title);
+  };
+
+  useEffect(() => {
+    queryForJobs();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This would actually return an array of the first 20 jobs. I'm not sure how it determines which 20 to show, however. It didn't have a ton of use, though. It basically just renders a list of jobs without the user needing to type anything in.

After speaking with Justin yesterday, we found that perhaps using the `InstantSearch` component from the `react-instantsearch-hooks-web` package would be a better option than the implementation here. Still working on updating the PR with those changes as testing is proving somewhat difficult.